Xamarin.Forms.Platform.Tizen/Native/Entry.cs

```diff
@@ -364,6 +385,26 @@ public virtual ESize Measure(int availableWidth, int availableHeight)
 
 		}
 
+		protected virtual void InvokeTextBlockFocused()
```

Comment:
Is there any special reason to provide it as a method?

Reply:
This method will be called in MEditor to trigger TextBlockFocused event, and it is needed for implementing MaterialPickerRenderer.

Reply:
why don't you invoke those events declared in `IEntry` instead?

Reply:
consider to apply [this pattern](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/events/how-to-raise-base-class-events-in-derived-classes)

Reply:
Fixed.